typescript/sdk/src/token/EvmERC20WarpModule.hardhat-test.ts

```diff
@@ -876,9 +877,7 @@ describe('EvmERC20WarpHyperlaneModule', async () => {
 
         await assertAllowedRebalancers(evmERC20WarpModule, expectedRebalancers);
       });
-    }
 
-    for (const tokenType of movableCollateralTypes) {
```

Comment:
why remove this?

Reply:
i have just grouped the tests that had the same loop and put them all together in a single loop

https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/14c325a19d53359f927aae3a0d3e00f07b419f37/typescript/sdk/src/token/EvmERC20WarpModule.hardhat-test.ts#L856-L1123